store/src/main/java/org/apache/rocketmq/store/ConsumeQueue.java

```diff
@@ -84,6 +84,58 @@ public boolean load() {
         return result;
     }
 
+    public void checkCommitLogAndConsumeQueueConsistent() {
+
+        String queueDir = this.storePath
```

Comment:
There is a helper to get queue dir: `StorePathConfigHelper#getStorePathConsumeQueue`